application/shared-kernel/InfrastructureCore/InfrastructureCoreConfiguration.cs

```diff
@@ -29,7 +30,13 @@ string connectionName
             ? Environment.GetEnvironmentVariable("DATABASE_CONNECTION_STRING")
             : builder.Configuration.GetConnectionString(connectionName);
         
-        builder.Services.AddSqlServer<T>(connectionString, optionsBuilder => { optionsBuilder.UseAzureSqlDefaults(); });
+        builder.AddSqlServerDbContext<T>(connectionName, null,
```

Comment:
This fails when running in Azure because there is no connection string configured for the `connectionName`.

This is the exception:

```
ConnectionString is missing. It should be provided in 'ConnectionStrings:account-management-database' or under the 'ConnectionString' key in 'Aspire:Microsoft:EntityFrameworkCore:SqlServer' or 'Aspire:Microsoft:EntityFrameworkCore:SqlServer:AccountManagementDbContext' configuration section. 
```

Reply:
I drop the PR until we know if we will have connection pooling in the future. And we will also investigate if it will give us limitations we apply multi-tenant.